x-pack/test/functional/services/monitoring/logstash_pipelines.js

```diff
@@ -64,6 +64,18 @@ export function MonitoringLogstashPipelinesProvider({ getService, getPageObjects
       }, []);
     }
 
+    async clickPipeline(id) {
+      const anchors = await testSubjects.findAll(SUBJ_PIPELINES_IDS);
+      for (let i = 0; i < anchors.length; i++) {
+        const anchor = anchors[i];
+        if ((await anchor.getVisibleText()) === id) {
+          return anchor.click();
+        }
+      }
+
+      throw new Error(`pipeline with id ${id} not found`);
```

Comment:
Just nit picking but, seems like this is the pipeline "name" rather than "id". Pretty sure pipelines get a unique id number as well as a string name.